iceoryx2/tests/service_event_tests.rs

```diff
@@ -490,30 +491,30 @@ mod service_event {
     }
 
     #[test]
-    // TODO iox2-139, enable when bitset is integrated into events
-    #[ignore]
     fn concurrent_reconnecting_listener_can_wait_for_triggering_notifiers<Sut: Service>() {
         let _watch_dog = Watchdog::new();
 
-        const NUMBER_OF_LISTENER_THREADS: usize = 2;
-        const NUMBER_OF_NOTIFIER_THREADS: usize = 2;
-        const NUMBER_OF_ITERATIONS: usize = 50;
-        const EVENT_ID: EventId = EventId::new(558);
+        let number_of_listener_threads =
+            (SystemInfo::NumberOfCpuCores.value() / 2).clamp(2, usize::MAX);
+        let number_of_notifier_threads =
+            (SystemInfo::NumberOfCpuCores.value() / 2).clamp(2, usize::MAX);
```

Comment:
I would double the number of thread instead of cutting them in halve ... or maybe do tests with both.